Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Proteinortho is a tool to detect orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0b/proteinortho-v6.0b.tar.gz"
+  sha256 "f59c436f6461f318ccebc1177a67eb70c29ef9ba222ad724de1fb2a769a120b5"
+  depends_on "cmake" => :build
+  depends_on "gcc"
```

Comment:
does it depend on gcc at runtime?

Reply:
It is it linked dynamically, that is why i thought i need that

Reply:
If there is Fortran in the sources, then `depends_on "gcc"` is the correct way to enable Fortran compilation. Dynamic linkage and seeing dependencies on gcc runtime libs when you build on your machine with, e.g., `export CC=gcc-8 ; cmake .. ; ... ` doesn't necessarily mean you have a true dependency on gcc.

TL;DR: Unless you are building Fortran sources, or have a legitimate reason that you can't build with Apple Clang, then I don't think you need the GCC dependency.